docs/pages/database-access/rbac.mdx

```diff
@@ -138,3 +155,235 @@ spec:
     db_users: ["{{internal.db_users}}"]
     db_names: ["{{internal.db_names}}"]
 ```
+
+## Database object import rules
```

Comment:
We should add a warning/note somewhere here saying that only Postgres is supported for now.

Reply:
Added to the top of the guide